tests/integ/modin/conftest.py

```diff
@@ -54,6 +55,17 @@ def setup_pandas_api_coverage_generator(pytestconfig):
         PandasAPICoverageGenerator()
 
 
+SKIP_SQL_COUNT_CHECK = False
+
```

Comment:
oh, does that needs to be a global variable? was that how we handle sql simplifier before?

Reply:
we also use a flag for sql simplifier, but it's implemented as a fixture. However, sql count check don't have access to a fixture, so I just use a global variable. It's indeed a global variable because I access it using `global` keyword later.